input/pagecontent/index.md

```diff
@@ -1,15 +1,86 @@
+
+Another terminology caveat is the difference between _Tags_ in HCS and in FHIR. In FHIR, `meta.tag` is a data structure for capturing general metadata and observations about a resource and is a collection of objects of type [`Coding`](https://build.fhir.org/datatypes.html#Coding). _Tags_ in HCS, however, are opaque values (equivalent to the `code` in a `Coding` data structure).
 
-The syntactic and semantic rules of the [HL7 Healthcare Privacy and Security Classification System (HCS), Release 1 (HCS)](http://www.hl7.org/implement/standards/product_brief.cfm?product_id=345) specify conceptually how to construct security labels in order to convey computable and interoperable privacy, security, and trust policies within a policy domain.
+### The Need for a FHIR DS4P Implementation Guide
+The [FHIR Security Labels Module](http://hl7.org/fhir/security-labels.html) defines the core mechanism for assigning security labels to a FHIR resource and specifies some [Core Security Labels](http://hl7.org/fhir/security-labels.html#core), such as _purpose of use_ and _confidentiality_ tag sets. This IG expands on the baseline provided by [FHIR Security Labels Module](http://hl7.org/fhir/security-labels.html), particularly in the following areas:
+
+- Differentiating which set of `Resource.meta.security` elements belongs to which policy in the (not uncommon) case where more than one policy applies to a resource, 
+
+- Designating the entity responsible for classifying the content with the assigned security label, 
+
+- Referencing related artifacts that are the basis for the policy being conveyed by the label, such as a patient privacy consent directive, the [trust contract](glossary.html#trust-contract) (which specifies agreed-upon system capabilities between the sender and the receiver), or the source of one or more provenance labels (e.g., FHIR Provenance resources),
+
+- Requiring that a _privacy mark_ be displayed to end users, and
+
+- Granular data segmentation at the sub-resource level, known as [inline labeling](inline.html).
+
+For example, defining the [`sec-label-basis` extension](StructureDefinition-extension-sec-label-basis.html) in this IG enables an implementer to group relevant security labels in `Resource.meta.security` into an HCS-conformant Security Labels, with  each applicable policy represented as a separate named tag set. For example, when the information in a resource is governed by both US _42 CFR Part 2_ for _substance use disorder_ confidentiality as well as _CFR 32 Part 2002_ for _Controlled Unclassified Information (CUI)_, the label assigned based on each of these policies will be distinguished using an instance of the [`sec-label-basis` extension](StructureDefinition-extension-sec-label-basis.html).
```

Comment:
bringing in just one example is confusing. I think there should be no example here. The bullets are clear enough at this level of the document.

Reply:
I agree. Seems like this very example is also given in `spec.md`